### ISSUE TITLE
Crazy colors on sent webcam photo

### ISSUE THREAD

**Issue by forteller** [wireapp/wire-webapp#1272]
Wire for Web Version: Couldn't find this by viewing the code in the dev tools. But I recently got a notification to update and did so.
Browser Version: Rambox Version: 0.5.3 (Electron: 1.4.15, Chromium: 53.0.2785.143, Node: 6.5.0) and Firefox 54.0b9 (64-bit)
OS: Ubuntu Gnome 16.10, x64

**What steps will reproduce the problem?**
1. Take a photo with built in webam in the app Cheese
2. Drag and drop into Wire or click on Add picture and choose it

**What is the expected result?**

Perfect replication of the original image

**What happens instead?**

Crazy colors!

![screenshot from 2017-05-23 16-44-49](https://cloud.githubusercontent.com/assets/343561/26360219/1c64d424-3fc7-11e7-8cae-743647953a23.png)

**Please provide any additional information below.**

Here is the original image: http://imgur.com/dhfszaG

I've tried to test this in these ways:
- Sending another pic taken in Cheese to the same person. Same result
- Sending the original pic to a different person. Same result
- Sending other pics, not taken with my webcam/Cheese. Good/Expected result
- Uploading the original pic to Imgur (see link above). Good/Expected result
- Sending a screenshot of the original pic (the screenshot aove). Good/Expected result!

I've not yet tried to take a pic with my built in webcam with another camera application other than Cheese. I can try this later if needed.

Thanks!

**Comment by bennycode** [comment]
Hi @forteller, thank you very much for reporting this issue. Your input is really valuable and helped us to reproduce the issue! 👍 We discovered that the problem exists within the image compression library ([Jimp](https://github.com/oliver-moran/jimp)) that we are using. 

We filed your issue on Jimp: https://github.com/oliver-moran/jimp/issues/292

Once your issue gets solved on Jimp, it will be solved here. 🥂
